verbit/streaming_client.py

```diff
@@ -47,6 +49,10 @@ class WebsocketStreamingClientSingleConnection:
     # events
     EVENT_EOS = 'EOS'
 
+    # endpoints
+    DEFAULT_WEBSOCKET_ENDPOINT = 'wss://speech.verbit.co/ws'
+    DEFAULT_AUTH_ENDPOINT = 'https://users.verbit.co/api/v1/auth'
```

Comment:
The only other option I can think of is to require passing this url into the client when creating it.
We can put it in all code examples and docs... It can be "sold" as a kind of preparation for having this service in different regions (so you need to pass a different URL to connect for EU for example).

This will not require any changes for customers already working with the code... it is the plus.
But it will require from new customers to put that in hard-coded.

WDYT?